backend/app/running_config/util/device_config_types.py

```diff
@@ -4,7 +4,7 @@
 from app.config_upload.util.netmiko_types import NetmikoDeviceType
 
 
-class XmlConfigConstants:
+class XmlConfigConstants(BaseModel):
```

Comment:
Nie wydaje mi się, że powinien to dziedziczyć po BaseModel. To są stałe, nie muszą być ani walidowane, ani niej jest to żaden DTO.